packages/endpoints-organizations-registrar/src/config/config.js

```diff
@@ -178,14 +178,14 @@ const createConfig = (packageJson) => {
     libS3Bucket: env.get('LIB_S3_BUCKET').required(!isTest).asString(),
     libUrl: env.get('LIB_URL').required(!isTest).asString(),
     custodiedDidWebHost: env.get('CUSTODIED_DID_WEB_HOST').asString(),
-    reminderDelayInMinutes: env
-      .get('REMINDER_DELAY_IN_MINUTES')
-      .default(10080)
-      .asIntPositive(),
     kmsPluginModule: env
       .get('KMS_PLUGIN_MODULE')
       .default('@velocitycareerlabs/db-kms')
       .asString(),
+    serviceConsentVersion: env
```

Comment:
should this be moveed to `signatoryConfig`?

Reply:
No this is a consent related to service creation, nothing to do with the signatory consent. I just saw during the refactor, that the version was also hardcoded in the business logic, so I moved it out to the config as well. @sloops77